src/geo/ui/legends/legends-view.js

```diff
@@ -109,7 +107,7 @@ var LegendsView = View.extend({
     this.$(this._container()).append(layerLegendsView.render().$el);
   },
 
-  _onLayerAddedOrRemoved: function (layerModel) {
+  _onLegendsChanged: function (layerModel) {
```

Comment:
I think I'd put a name related with layers, like `onLayersChanged`, because this is the layer events' callback.

Reply:
wow, that was a fail 🤦‍♂️